telegram/ext/_application.py

```diff
@@ -338,8 +336,7 @@ def concurrent_updates(self) -> int:
         """:obj:`int`: The number of concurrent updates that will be processed in parallel. A
         value of ``0`` indicates updates are *not* being processed concurrently.
 
-        .. seealso:: `Concurrency <https://github.com/\
-            python-telegram-bot/python-telegram-bot/wiki/Concurrency>`_
+        .. seealso:: :wiki:`Concurrency`
```

Comment:
If the link text is the same as the relative address, the syntax of `:wiki:` link becomes even more concise